views/client.go

```diff
@@ -187,8 +187,8 @@ func viewershipEventsToMetrics(rows []ViewershipEventRow, spec QuerySpec) []Metr
 			Subdivision:      toStringPtr(row.Subdivision, spec.hasBreakdownBy("subdivision")),
 			TimeZone:         toStringPtr(row.TimeZone, spec.hasBreakdownBy("timezone")),
 			GeoHash:          toStringPtr(row.GeoHash, spec.hasBreakdownBy("geohash")),
-			ViewCount:        row.ViewCount,
-			PlaytimeMins:     row.PlaytimeMins,
+			ViewCount:        toInt64Ptr(row.ViewCount, spec.Detailed),
+			PlaytimeMins:     toFloat64Ptr(row.PlaytimeMins, spec.Detailed),
```

Comment:
These 2 fields are actually not detailed, they belong to the basic viewership metrics and should always be in the response. So they are always "asked".
```suggestion
			ViewCount:        toInt64Ptr(row.ViewCount, true),
			PlaytimeMins:     toFloat64Ptr(row.PlaytimeMins, true),
```

But thinking about this, we probably don't want to return `null` on the API responses for these fields, as we shouldn't leak the datasource bug to the end-user. So IMO these fields should always be present and instead we return `0` if they are `null`, WDYT? We could fix this directly [on the SQL queries](https://github.com/livepeer/livepeer-data/blob/6f19bae9c8b185dfaebaf81ff9b0840385f3d233/views/bigquery.go#L157-L158), like adding a [`COALESCE`](https://cloud.google.com/bigquery/docs/reference/standard-sql/conditional_expressions#coalesce) on those fields, or we could just have a logic here to check `v == null ? 0 : v`.